python/nano/benchmark/run.py

```diff
@@ -43,11 +46,41 @@ def run(self):
     @staticmethod
     def _extract_logs(output: str):
         logs = [match.group(1) for match in re.finditer(">>>(.*?)<<<", output)]
+        logs = [json.loads(log) for log in logs]
         return logs
 
     def _save_logs(self, logs: list):
+        is_pr = True if os.environ.get('IS_PR') is not None else False
+        timestamp = datetime.now()
+        conn = psycopg2.connect(
```

Comment:
@glorysdj we are using k8s secrets to connect to a database deployed on k8s, do you think it is safe to do this in github action?

Reply:
Or should we use the secrets in github action https://github.com/Azure/actions-workflow-samples/blob/master/assets/create-secrets-for-GitHub-workflows.md